index.html

```diff
@@ -23,22 +23,147 @@ <h1>Hey There.i'm Said</h1>
-        <li><a href="https://twitter.com/saidlaasr">
-          <img src="img/win.svg" alt="win">
-        </a></li>
-        <li><a href="https://twitter.com/saidlaasr">
-          <img src="img/twitter.svg" alt="twitter">
-        </a></li>
-        <li><a href="https://twitter.com/saidlaasr">
-          <img src="img/Vector.svg" alt="victor">
-        </a></li>
+        <li>
+          <a href="https://github.com/Said-laasri">
+            <img src="img/github.svg" alt="github">
+          </a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/said-laasri-8a4367172/">
+            <img src="img/linkdin.svg" alt="linkdin">
+          </a>
+        </li>
+        <li>
+          <a href="https://twitter.com/saidlaasr">
+            <img src="img/win.svg" alt="win">
+          </a>
+        </li>
+        <li>
+          <a href="https://twitter.com/saidlaasr">
+            <img src="img/twitter.svg" alt="twitter">
+          </a>
+        </li>
+        <li>
+          <a href="https://twitter.com/saidlaasr">
+            <img src="img/Vector.svg" alt="victor">
+          </a>
+        </li>
       </ul>
     </section>
+    <section class="work">
+      <h1 class="title2">My Recent Works</h1>
+      <section class="work project0">
+        <div class="info-project">
+          <h2>Multi-Post Stories</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <ul class="tags">
+          <li class="css">css</li>
+          <li class="html">html</li>
+          <li class="boot">bootstrap</li>
+          <li class="ruby">Ruby</li>
+        </ul>
+        <form class="check">
+          <button type="button" class="click">See Project</button>
+        </form>
+      </section>
+      <section class="work project1">
+        <div class="linfo">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="lang">
+            <li class="ht">html</li>
+            <li class="bo">bootstrap</li>
+            <li class="ru">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="but">See Project</button>
+        </form>
+      </section>
+      <section class="work projectq">
+        <div class="linfoq">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="langq">
+            <li class="htq">html</li>
+            <li class="boq">bootstrap</li>
+            <li class="ruq">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="butq">See Project</button>
```

Comment:
- [ ] Kindly remove the black border in the `See Project` buttons as in the Figma design.
- [ ] You should also use bold font-weight for the button text as in the design.
#### Imple
![image](https://user-images.githubusercontent.com/44057863/170738203-731613c1-d977-4eb6-8759-6c2b97c387ff.png)
#### Design
![image](https://user-images.githubusercontent.com/44057863/170738371-7417d4fe-bd6b-415a-be6b-ffffb00e0b64.png)

Reply:
done